firmware/src/src/CRSF/crsf_protocol.h

```diff
@@ -61,22 +61,43 @@ typedef enum {
       0x7C,  // write with 8 byte chunked binary (OpenTX outbound telemetry buffer limit)
 } crsf_frame_type_e;
 
-typedef enum {
-  CRSF_ADDRESS_BROADCAST = 0x00,
-  CRSF_ADDRESS_USB = 0x10,
-  CRSF_ADDRESS_TBS_CORE_PNP_PRO = 0x80,
-  CRSF_ADDRESS_RESERVED1 = 0x8A,
-  CRSF_ADDRESS_CURRENT_SENSOR = 0xC0,
-  CRSF_ADDRESS_GPS = 0xC2,
-  CRSF_ADDRESS_TBS_BLACKBOX = 0xC4,
-  CRSF_ADDRESS_FLIGHT_CONTROLLER = 0xC8,
-  CRSF_ADDRESS_RESERVED2 = 0xCA,
-  CRSF_ADDRESS_RACE_TAG = 0xCC,
-  CRSF_ADDRESS_RADIO_TRANSMITTER = 0xEA,
-  CRSF_ADDRESS_CRSF_RECEIVER = 0xEC,
-  CRSF_ADDRESS_CRSF_TRANSMITTER = 0xEE,
+typedef enum
+{
+    CRSF_ADDRESS_BROADCAST = 0x00,
+    CRSF_ADDRESS_USB = 0x10,
+    CRSF_ADDRESS_TBS_CORE_PNP_PRO = 0x80,
+    CRSF_ADDRESS_RESERVED1 = 0x8A,
+    CRSF_ADDRESS_CURRENT_SENSOR = 0xC0,
+    CRSF_ADDRESS_GPS = 0xC2,
+    CRSF_ADDRESS_TBS_BLACKBOX = 0xC4,
+    CRSF_ADDRESS_FLIGHT_CONTROLLER = 0xC8,
+    CRSF_ADDRESS_RESERVED2 = 0xCA,
+    CRSF_ADDRESS_RACE_TAG = 0xCC,
+    CRSF_ADDRESS_RADIO_TRANSMITTER = 0xEA,
+    CRSF_ADDRESS_CRSF_RECEIVER = 0xEC,
+    CRSF_ADDRESS_CRSF_TRANSMITTER = 0xEE,
+    CRSF_ADDRESS_ELRS_LUA = 0xEF
 } crsf_addr_e;
 
+// Express LRS 3.0
+typedef enum : uint8_t
```

Comment:
TBS Crossfire RFModes are 0(4Hz), 1(50Hz), and 2(150Hz). The one here are specific to ELRS and do not match the original CRSF modes.

Reply:
In the process of trying to figure out how CRSF really works internally. I assume there is going to be some other way to tell if it's ELRS or a Crossfire system that I can incorporate. As of now it only is used to show the mode in the Log so should still work for both. Open to help here :)
There also doesn't need to be CRSF in & out files. I'm sure a proper library could be used for both.